lib/tasks/reactions.rake

```diff
@@ -50,3 +50,54 @@ task "reactions:generate", [:post_id, :reactions_count, :reaction] => [:environm
       .toggle!
   end
 end
+
+desc "Converts reactions to like"
+task "reactions:nuke", [:reaction_list_to_convert] => [:environment] do |_, args|
+  puts "Disabling the discourse_reactions plugin"
```

Comment:
we need a prompt confirmation here

Reply:
May I add a variable like `:disable_plugin` if the value is provided then we will consider the value, and if not provided disable the plugin?

Reply:
no, what I mean was we need a confirmation at the start of the task.

"You are about to destroy all reactions from database, are you sure ? y/n"

Reply:
Okay I got it😅